plugins/woocommerce/src/Utilities/PluginUtil.php

```diff
@@ -171,4 +171,65 @@ private function handle_plugin_de_activation(): void {
 		$this->woocommerce_aware_plugins        = null;
 		$this->woocommerce_aware_active_plugins = null;
 	}
+
+	/**
+	 * Util function to generate warning string for incompatible features based on active plugins.
+	 *
+	 * @param string $feature_id Feature id.
+	 * @param array  $plugin_feature_info Array of plugin feature info. See FeaturesControllers->get_compatible_plugins_for_feature() for details.
+	 *
+	 * @return string Warning string.
+	 */
+	public function generate_incompatible_plugin_feature_warning( string $feature_id, array $plugin_feature_info ) : string {
+		$feature_warning    = '';
+		$incompatibles      = array_merge( $plugin_feature_info['incompatible'], $plugin_feature_info['uncertain'] );
+		$incompatibles      = array_filter( $incompatibles, 'is_plugin_active' );
+		$incompatible_count = count( $incompatibles );
+		if ( $incompatible_count > 0 ) {
+			if ( 1 === $incompatible_count ) {
+				/* translators: %s = printable plugin name */
+				$feature_warning = sprintf( __( '⚠ 1 Incompatible plugin detected (%s).', 'woocommerce' ), $this->get_plugin_name( $incompatibles[0] ) );
+			} elseif ( 2 === $incompatible_count ) {
+				$feature_warning = sprintf(
+				/* translators: %1\$s, %2\$s = printable plugin names */
```

Comment:
```suggestion
					/* translators: %1\$s, %2\$s = printable plugin names */
```

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/38993/commits/94fce9c169809e484a9d75803600ae3ad88b4ee5